### DISCUSSION TITLE
torch.cuda.is_available() returns False on Intel GPU

### DISCUSSION THREAD

**Discussion by DmitrySvit** [vosen/ZLUDA#199] [Q&A]
I'm using PyTorch with ZLUDA to enable CUDA compatibility on my Intel GPU, but I'm encountering an issue where torch.cuda.is_available() returns False, indicating that CUDA isn't recognized by PyTorch despite ZLUDA's presence. This is preventing me from leveraging GPU acceleration in my PyTorch models. Has anyone encountered this issue and found a workaround or solution to ensure PyTorch recognizes the CUDA emulation provided by ZLUDA?

**Comment by vosen** [comment]
Intel GPU? But ZLUDA only supports AMD GPU

sob., 6 kwi 2024, 12:52 użytkownik DmitrySvit ***@***.***>
napisał:

**Reply by DmitrySvit** [reply]
Intel GPU support was only removed in the latest third version. Before that it was there. I used the second version, but there was no difference.

**Reply by vosen** [reply]
Old Intel version is too incomplete to run PyTorch
